user_templates/mcu.cfg

```diff
@@ -13,3 +13,28 @@
 
 # ------------------------------------------------------------------------------------------
 
+
+#-------------------------#
+#   TMCs DRIVERS          #
+#-------------------------#
+
+# ---------------------------------------------------------------------- AXIS DRIVERS ----> optional, but only one line need to be selected
+### 1. X Drivers -------------------------------------------------------------------------
+# [include config/hardware/axis/X/TMC/TMC2209.cfg]
+# [include config/hardware/axis/X/TMC/TMC2209_V0specific.cfg]
+### 2. Y Drivers -------------------------------------------------------------------------
+# [include config/hardware/axis/Y/TMC/TMC2209.cfg]
+# [include config/hardware/axis/Y/TMC/TMC2209_V0specific.cfg]
```

Comment:
i would suggest to add a comment behind the V0 specific, since it is only meant for early LDO releases of the v0.1